app/scripts/background.ts

```diff
@@ -34,24 +30,59 @@ const onClickContextMenus = async (
       type: 'basic',
       title: chrome.i18n.getMessage('registrationSuccessTitle'),
       message: chrome.i18n.getMessage('registrationSuccessBody', [emojiName, teamName]),
-      iconUrl: info.srcUrl
+      iconUrl: imageUrl,
     })
+    return { teamName, emojiName, imageUrl, sessionInfo }
```

Comment:
Please move this line to after `try/catch`.

This catch statement is for non-chrome browsers.
So, Communication with Slack is successful in `catch` statement too.

Reply:
I had misunderstood.

Reply:
fixed.